src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -93,11 +93,15 @@
                             <div class="tab-pane" id="python-help">
                                 <ol>
                                     <li>
-                                        Generate wheel for the package.
-                                        <code>python setup.py bdist_wheel -d [dest_folder]</code>
+                                        Install apistubgen.
```

Comment:
Why are we undoing this part considering the python path is being fixed in the same PR?

Reply:
I would like to change the text only after fix is verified on deployed app. So far it fails with weird error on deployed app. I will submit another PR to fix the help text once everything is confirmed good to go.